mingw-w64-x86_64.cmake

```diff
@@ -0,0 +1,26 @@
+# Sample toolchain file for building for Windows from an Ubuntu Linux system.
+#
+# Typical usage:
+#    *) install cross compiler: `sudo apt-get install mingw-w64`
+#    *) cd build
+#    *) cmake -DCMAKE_TOOLCHAIN_FILE=~/mingw-w64-x86_64.cmake ..
+
+set(CMAKE_SYSTEM_NAME Windows)
+set(TOOLCHAIN_PREFIX x86_64-w64-mingw32)
+
+# cross compilers to use for C, C++ and Fortran
+set(CMAKE_C_COMPILER ${TOOLCHAIN_PREFIX}-gcc)
+set(CMAKE_CXX_COMPILER ${TOOLCHAIN_PREFIX}-g++-posix)
```

Comment:
Any reason this is using `-posix` over the base?

Reply:
cpr doesn't compile with win32, "base" on ubuntu is -win32. And in OHL std::thread is not defined in win32 mingw.

Reply:
I switched these to  target link options for OHL.